core/src/main/java/de/danoeh/antennapod/core/storage/PodDBAdapter.java

```diff
@@ -188,7 +189,8 @@
             + KEY_IS_PAGED + " INTEGER DEFAULT 0,"
             + KEY_NEXT_PAGE_LINK + " TEXT,"
             + KEY_HIDE + " TEXT,"
-            + KEY_LAST_UPDATE_FAILED + " INTEGER DEFAULT 0)";
+            + KEY_LAST_UPDATE_FAILED + " INTEGER DEFAULT 0,"
+            + KEY_AUTO_DELETE_ACTION + " INTEGER DEFAULT 0)";
```

Comment:
You do not upgrade the existing table, do you? Have a look at StorageCallbacksImpl.java (increment the version, ``if(oldVersion <= 16) { ...`` and add the column)
Everything looks good and ready to be pulled, but this addition is necessary